samples/cpp/benchmark_app/remote_blobs_filling.cpp

```diff
@@ -2,15 +2,21 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include "remote_blobs_filling.hpp"
+
 #include <memory>
 #include <random>
+#include <samples/slog.hpp>
```

Comment:
Mix of headers

Reply:
c-lang checker is changing this. I've move the header, but it might change it again.